### ISSUE TITLE
Rope is open for adoption

### ISSUE THREAD

**Issue by mcepl** [python-rope/rope#360]
It seems to me that I just don't have any time to be the only who cares for this zombie (reanimated corps) any more, so I have to give up on my efforts to keep it alive. Will you step up, or should we just close the shop?

CC: @aligrudi, @emacsway, @joystein, @soupytwist, @thomasjm, @tommclaughlin

**Comment by lieryan** [comment]
:hand: First thing first, thanks for maintaining this project for all this time. I'd be willing to assist in any ways I can to keep the project going.

**Comment by mcepl** [comment]
You have been invited.

**Comment by lieryan** [comment]
I've accepted the invite. Thanks again @mcepl and all other prior maintainers for this great project.

In addition to looking reviewing merge requests and looking into the issue trackers, I also intend to look into ways to document and make it easier to use Rope in text editors/IDEs and contribute to Rope, as well as advertising about Rope in various developer communities which allows open source projects advertisements.

**Comment by soupytwist** [comment]
Thank you, and thanks Matej for supporting this for so long. I haven't been able to devote time to it for a while now.

**Comment by climbus** [comment]
Hi, 
Python hasn't any good refactoring tool. Pycharm has poor refactoring options, VSCode by LSP too. Vim rope plugin is dead.
By now, only Rope is option to provide good tool for refactorings. I hope it'll be developed.  I can help, but not with documentation because of my English. Especially, I can help with testing editors support. 
If you have some entry task, for example adding some unit tests, please send me an issue with short description.

**Comment by mcepl** [comment]
> Vim rope plugin is dead.

That is not correct … https://github.com/python-rope/ropevim is as alive as ever.

**Comment by climbus** [comment]
Good to hear. I'll check it. 

Could you look on my pull request in rope-mode project? https://github.com/python-rope/ropemode/pull/17

**Comment by lieryan** [comment]
> If you have some entry task, for example adding some unit tests

Hi @climbus, thank you for your interest in rope. 

I'm currently grooming our issues backlog. One of the goal of backlog grooming is to identify tickets that can be ["good first issue"](https://github.com/python-rope/rope/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) and to refine the ticket descriptions to make sure there is enough information for contributors to pick up.

I don't currently have much other than documentation tickets yet, but if you come back in a couple days or by next week, I'll make sure to identify more tickets as "good first issue".

Of course you're always welcome to work on any tickets that you feel interest you in the mean time! Or even to write your own tickets for something you're interested in looking into. I think Open source works best if you are scratching your own itch, rather than being assigned work by someone else like you would in a workplace. 

As I'm a rather new maintainer, I don't really know everything around here yet, but I'll try to accommodate and provide any assistance that I can to point you to the right directions.
